src/test/resources/pacts/publicapi-connector-get-payment-with-awaiting-capture-request-state-and-gateway-transaction-id.json

```diff
@@ -7,13 +7,14 @@
   },
   "interactions": [
     {
-      "description": "get one charge in awaiting capture request status",
+      "description": "get one charge in awaiting capture request status with gateway transaction id",
       "providerStates": [
         {
           "name": "a charge with delayed capture true and awaiting capture request status exists",
           "params": {
             "gateway_account_id": "123456",
-            "charge_id": "ch_123abc456def"
+            "charge_id": "ch_123abc456def",
```

Comment:
the provider state could be renamed to "a charge exists" (new method on connector). the current name "a charge with delayed capture true and awaiting capture request status exists" is weird because we aren't setting delayed capture to true in the params, nor is the request status being set!

Reply:
Done!